src/pages/edge/adobe-journey-optimizer/tabs/api-reference.md

```diff
@@ -74,25 +74,25 @@ public static void handleNotificationResponse(final Intent intent,
 
 ```swift
 static func handleNotificationResponse(_ response: UNNotificationResponse,
```

Comment:
Can we still keep the old API with deprecated / removed warning? We decided to do it for other APIs like registerExtension which will be removed with Android 3.0.0.